docs/settings.rst

```diff
@@ -240,3 +235,23 @@ Default: ``False``
    GRAPHENE = {
       'GRAPHIQL_SHOULD_PERSIST_HEADERS': False,
    }
+
+
+``GRAPHIQL_INPUT_VALUE_DEPRECATION``
+------------------------------------
+
+Set to ``True`` if you want GraphiQL to display a deprecation section on field doc view.
```

Comment:
I _think_ I know what this means, but I'm not actually positive. I think it would be worth giving an example of a graphene/graphene-django schema definition for when this would come into play and what the effect of setting it to `True` is.

Reply:
I've added an example and some more details on what the setting actually does.